l10n_fr_ecotaxe/models/ecotaxe_line_product.py

```diff
@@ -28,18 +29,15 @@ class EcotaxeLineProduct(models.Model):
     amount = fields.Float(
         digits="Ecotaxe",
         compute="_compute_ecotaxe",
-        help="Ecotaxe Amount computed form Classification or forced ecotaxe amount",
+        help="Ecotaxe Amount computed form Classification or " "forced ecotaxe amount",
         store=True,
     )
     display_name = fields.Char(compute="_compute_display_name")
 
     @api.depends("classification_id", "amount")
     def _compute_display_name(self):
         for rec in self:
-            rec.display_name = "%s (%s)" % (
-                rec.classification_id.name,
-                rec.amount,
-            )
+            rec.display_name = f"{rec.classification_id.name} {rec.amount}"
```

Comment:
Is it wanted to remove the `()` ?
I would have done 
```f"{rec.classification_id.name} ({rec.amount})"```

Reply:
We have removed the `()` to dismiss `%s` string formatting as it's an `pre-commit` error now. There was no `()` originally behind the `amount`